src_tests/unittests/osm_configurator/view/states/test_state.py

```diff
@@ -2,31 +2,32 @@
 
 import src.osm_configurator.view.states.state as state_i
 import src.osm_configurator.view.states.positioned_frame as positioned_frame_i
-import src.osm_configurator.view.toplevelframes.top_level_frame as top_level_frame_i
 import src.osm_configurator.view.states.state_manager as state_manager_i
 import src.osm_configurator.view.states.state_name_enum as state_name_enum_i
 
+import src_tests.unittests.osm_configurator.view.states.top_level_frame_stub as top_level_frame_stub_i
```

Comment:
Why use stub here and not the other stubs?

Reply:
A new stub was defined, so states can be testen independely from other things, because it only wants a TopLevel frame and not stateManager or mainWindos or whatever anymore, this makes it easy to tets ist, since we are not testing the frame here, we are testing the state